server/session.go

```diff
@@ -451,8 +451,8 @@ func (s *Session) dispatch(msg *ClientComMessage) {
 
 // Request to subscribe to a topic
 func (s *Session) subscribe(msg *ClientComMessage) {
-	if strings.HasPrefix(msg.Original, "new") {
-		// Request to create a new named topic.
+	if strings.HasPrefix(msg.Original, "new") || strings.HasPrefix(msg.Original, "nch") {
```

Comment:
Consider making this condition a utility function.
It's also present at init_topic.go:41.

Reply:
It's only two places. I don't expect more of them. I'll add it later if needed.